src/shogun/base/SGObject.h

```diff
@@ -315,6 +444,35 @@ class CSGObject
 	 */
 	virtual void save_serializable_post() throw (ShogunException);
 
+	/** Sets a value for any parameter which is identified by a tag.
```

Comment:
This is protected, so the docs are different.

You should tell the developer here that he can use this method to register member variables for access through get/set and serialization